PacFactory.js

```diff
@@ -16,7 +16,7 @@ function makePac(){
   let velocity = getRandom(20);
   let position = getRandom(270);
   let container = document.getElementById('container');
-  let newPac = document.createElement('pac');
```

Comment:
Since `pac` isn't a registered custom element, it doesn't know how to render the `src` attribute you stated on line:60 in the browser.